tools/mo/openvino/tools/mo/moc_frontend/check_config.py

```diff
@@ -33,11 +51,16 @@ def legacy_extensions_used(argv: argparse.Namespace):
 
 def new_extensions_used(argv: argparse.Namespace):
     if any_extensions_used(argv):
-        extensions = argv.extensions.split(',')
+        extensions = argv.extensions
+        if not isinstance(extensions, list):
+            extensions = [extensions]
         new_ext_counter = 0
-        for extension in argv.extensions.split(','):
-            path = Path(extension)
-            if path.is_file() and (path.suffix == '.so' or path.suffix == '.dll'):
+        for extension in extensions:
+            if isinstance(extension, str):
+                path = Path(extension)
+                if path.is_file() and (path.suffix == '.so' or path.suffix == '.dll'):
+                    new_ext_counter += 1
+            else:
                 new_ext_counter += 1
         if new_ext_counter == len(extensions):
             return True # provided only new extensions
```

Comment:
Same as above

Reply:
If extension is not string we consider it as extension of new type, there is no error in it. The error should occur only if both legacy and new extensions are used.